testing/util/sync_aggregate.go

```diff
@@ -1,73 +0,0 @@
-package util
-
-import (
-	"github.com/pkg/errors"
-	"github.com/prysmaticlabs/go-bitfield"
-	"github.com/prysmaticlabs/prysm/beacon-chain/core/signing"
-	p2pType "github.com/prysmaticlabs/prysm/beacon-chain/p2p/types"
-	"github.com/prysmaticlabs/prysm/beacon-chain/state"
-	"github.com/prysmaticlabs/prysm/config/params"
-	"github.com/prysmaticlabs/prysm/crypto/bls"
-	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
-	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
-	"github.com/prysmaticlabs/prysm/time/slots"
-)
-
-func generateSyncAggregate(st state.BeaconState, privs []bls.SecretKey, parentRoot [32]byte) (*ethpb.SyncAggregate, error) {
```

Comment:
This function may be salvaged, the signature here is wrong and is never really used. Perhaps it can be saved in a way that it would sign correctly on fork transition, but since it's not used I figured it was better removed.

Reply:
I had to reuse it because it turned out to be used by the monitor indirectly in tests